aviary/variable_info/functions.py

```diff
@@ -144,13 +145,17 @@ def name_filter(name):
             group.promotes(comp.name, inputs=in_var_names, outputs=comp_promoted_outputs)
 
     if overridden_outputs:
-        print("\nThe following variables have been overridden:")
+        if aviary_inputs.get_val(Settings.VERBOSITY).value >= 1:
+            print("\nThe following variables have been overridden:")
         for prom_name in sorted(overridden_outputs):
-            print(f"  '{prom_name}")
+            item = aviary_inputs.get_item(prom_name)
+            print(f"  '{prom_name}  {item[0]}  {item[1]}")
```

Comment:
```suggestion
            val, units= aviary_inputs.get_item(prom_name)
            print(f"  '{prom_name}  {val}  {units}")
```
slightly more clear what is happening, not a big deal either way

Reply:
Agree.